Issue with spark support in 3.12 docker image

**Description**
The docker image for 0.3.12-2.11-spark2.4 works for non-spark notebooks, but fails with spark notebooks with the following error:

```[ERROR]   (Logged from NotebookSession.scala:174)
   |     java.io.IOException: Cannot run program "spark-submit": error=2, No such file or directory
   |     java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
   |     polynote.kernel.remote.SocketTransport$DeploySubprocess$$anonfun$deployKernel$1$$anonfun$apply$65$$anonfun$apply$66$$anonfun$apply$67$$anonfun$apply$68$$anonfun$apply$69.apply(transport.scala:347)
   |     polynote.kernel.remote.SocketTransport$DeploySubprocess$$anonfun$deployKernel$1$$anonfun$apply$65$$anonfun$apply$66$$anonfun$apply$67$$anonfun$apply$68$$anonfun$apply$69.apply(transport.scala:347)
   |     zio.internal.FiberContext.evaluateNow(FiberContext.scala:458)
   |     zio.internal.FiberContext.zio$internal$FiberContext$$run$body$2(FiberContext.scala:687)
   |     zio.internal.FiberContext$$anonfun$12.run(FiberContext.scala:687)
   |     java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
   |     java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
   |     java.lang.Thread.run(Thread.java:748)
```

**Reproduction**
Steps to reproduce the behavior:
1. Run docker image 0.3.12-2.11-spark2.4 without spark - works fine
2. Test with spark_master set to local[*] in spark config - fails with Kernel error as above
3. Try same process with previous version docker image (0.3.12-2.11-spark2.4) - works fine

**Environment**
Are you using the official Docker distribution? If so, please provide your `docker run` command and any other details that may be relevant (mount points, etc). 

```
export POLYNOTE_VERSION="0.3.11-2.11-spark2.4"
docker run --name polynote -it -p 127.0.0.1:8192:8192 -p 127.0.0.1:4040-4050:4040-4050 -v `pwd`/polynote/config.yml:/opt/config/config.yml -v /Users/username/projects/_notebook_repos:/opt/notebooks/ -v /Users/username/.ivy2/local:/home/polly/.ivy2/local polynote/polynote:$POLYNOTE_VERSION --config /opt/config/config.yml
```

Otherwise, please provide the following information:
- Polynote Version: 
   - Example: 0.2.12
- Operating System: 
  - Example: Mac OS Mojave 10.14.6
- JVM Version:
  - Example: 1.8.0_212
- Python Version:
  - Example: 2.7.15
- How you installed Polynote: 
  - Example: I downloaded this release https://github.com/polynote/polynote/releases/download/0.2.12/polynote-dist.tar.gz and then followed [these instructions](https://polynote.org/docs/01-installation.html).
- The command you are using to start Polynote:
  - Example: `./polynote.py`
- Any other details that might be useful, such as whether you are running Polynote as a service, using a custom script, etc.  

**Configuration**
If applicable, please attach or paste your Polynote configuration (`config.yml`)

**Notebook**
If applicable, please attach or paste the notebook file (`.ipynb`) you were trying to run when you see this bug. 

**Logs**
If applicable, please attach or paste the Polynote log

**Screenshots**
If applicable, please attach screenshots to help explain your problem.

**Additional context**
Add any other context about the problem here.

--- a/docker/spark-2.4/install_spark.sh
+++ b/docker/spark-2.4/install_spark.sh
@@ -22,7 +22,7 @@ else
 fi
 
 pushd /opt
-wget -q "https://downloads.apache.org/spark/${SPARK_VERSION_DIR}/${SPARK_NAME}.tgz" \
+wget -q "https://archive.apache.org/dist/spark/${SPARK_VERSION_DIR}/${SPARK_NAME}.tgz" \
   && tar zxpf "${SPARK_NAME}.tgz" \
   && mv "${SPARK_NAME}" spark \
   && rm "${SPARK_NAME}.tgz"

